geometry_msgs/PoseStamped target
std_msgs/Float64 x_offset
std_msgs/Float64 y_offset
---
geometry_msgs/Pose goal